app/src/main/res/values-b+pt/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">SpeakOut</string>
+    <string name="content_description_app_logo">Logo da aplicação.</string>
+
+    <!--  Login Activity  -->
+    <string name="login_success_activity_login">Login realizado com sucesso.</string>
+    <string name="edittext_email_activity_login">Endereço de email</string>
+    <string name="edittext_password_activity_login">Senha</string>
+    <string name="textview_signup_activity_login">Ainda não cadastrado? Criar uma conta.</string>
+    <string name="textview_forgot_password_activity_login">Esqueçeu a senha? Recuperar.</string>
+    <string name="button_login_activity_login">Concluir</string>
+
```

Comment:
Split the strings in regions, kind of a good practice to became easy to search for something.